doc/troubleshooting.md

```diff
@@ -88,8 +88,8 @@ be deleted by Deployment controller.
 
 ## External-IP details truncated in older Kubectl Client Versions
 
-If we enable external access while deploying a Pravega Cluster, an external IP is assigned to the Pravega Controller endpoint, which is used by users to access it. These external IP details can be viewed in the output of the `kubectl get svc`. 
-However, if we use an older Kubectl client version `v1.10.x` or lower, the external IP details in the output of the `kubectl get svc` are truncated.
+When pravega is deployed with "external-access enabled", an external IP is assigned to the Pravega Controller and Segment Store services, which is used by clients to access it. These external IP details can be viewed in the output of the `kubectl get svc`. 
+However, when using kubectl client version `v1.10.x` or lower, the External IP for the Controller and Segment Store services appears truncated in the output.
```

Comment:
Issue mentions kubectl version `v1.10.5`. Not sure if this also manifests in other kubectl client versions. It would be good to stick to the version number mentioned in the issue unless you have tried reproducing with other version numbers.

Reply:
I have tested it with the relevant version numbers before mentioning them